### DISCUSSION TITLE
vector database migration to another dify instance

### DISCUSSION THREAD

**Discussion by ddnoodle** [langgenius/dify#6999] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hi, 
We have two Dify instances installed using local dockers, one is for development environment and the other one is for production environment. The Dify version and LLMs are identical to the two env.

App is built and tested in dev env and later to be published in production env.

There are some large knowledge base which includes a large number of files (500+). It tooks much time to make it correctly uploaded and processed on dev env.
When delivering the related app on production env, it is time consuming to upload/embedding those files again in product env.
Just want to know if there is a way/tool to migrate the vector db on Dev env to Prod env.
If not, should this be a potential feature request?

**Comment by Gaocarri** [comment]
maybe you can use "/datasets?category=api" to write a script, then use script to migrate it

**Reply by xqiu** [reply]
seems it's not available in cloud.dify.ai?   https://cloud.dify.ai/datasets?category=api

**Reply by ddnoodle** [reply]
> maybe you can use "/datasets?category=api" to write a script, then use script to migrate it
Thanks, I checked the API, it is not what I wanted. The API is a api-version of web-based knowledge dataset operation.
what I want is to clone the specific existing knowledge dataset from one instance to another.
